motoman_driver/launch/io_relay.launch

```diff
@@ -0,0 +1,12 @@
+<!-- 
+  Wrapper launch file for I/O relay node.
+-->
+<launch>
+	<!-- IP of robot (or PC running simulation) -->
+	<arg name="robot_ip" doc="IP of controller" />
+
+	<!-- put them on the parameter server -->
+	<param name="robot_ip_address" type="str" value="$(arg robot_ip)" />
+
+        <node name="io_relay" pkg="motoman_driver" type="io_relay" />
+</launch>
```

Comment:
Please check indentation in this file: 2-spaces per level, no tabs.

Reply:
Addressed in f0834eb